### ISSUE TITLE
"Log out" button at "Me" page doesn't have accessible colors

### ISSUE THREAD

**Issue by simison** [Automattic/wp-calypso#49179]
"Log out" button at `/me`:

<img width="204" alt="image" src="https://user-images.githubusercontent.com/87168/105480718-c2f78b80-5cae-11eb-8105-18015af0785f.png">

https://webaim.org/resources/contrastchecker/?fcolor=74787F&bcolor=FFFFFF

<img width="740" alt="image" src="https://user-images.githubusercontent.com/87168/105480733-c9860300-5cae-11eb-8f4d-ba90befd9ac7.png">

**Comment by getdave** [comment]
Thanks for reporting.

**Comment by Aurorum** [comment]
Hi @simison, which colour scheme are you using in that screenshot? I've tried a couple, but they all seem to pass (as `--color-text-subtle` on white always should).


Classic Dark:
<img width="745" alt="Screenshot 2021-02-04 at 11 01 04" src="https://user-images.githubusercontent.com/43215253/106883745-4e721300-66d8-11eb-811e-a2c6ff770730.png">

**Comment by retrofox** [comment]
It seems to get the proper contrast to me. The computed color is `rgb(100,105,112)`...

<img width="1792" alt="Screen Shot 2021-03-30 at 9 56 06 AM" src="https://user-images.githubusercontent.com/77539/112992088-3f3e9d00-913e-11eb-9063-dd0c126c48fa.png">

that matches with what @Aurorum pointed us above `#646970`.
With these values, [the contract checker is ok](https://webaim.org/resources/contrastchecker/?fcolor=74787F&bcolor=FFFFFF).

Let me close this issue, and please feel free to re-open it again whether you still consider it appropriate.
